index.html

```diff
@@ -122,7 +122,7 @@ <h2>Multi_Post Stories</h2>
   </ul>
 </div>
   <p class="Mobile_vesion_list">A daily selection of privately presonalizaed reads: no accounts or singn-up required.</p>
-  <p class="">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+  <p class="Desktop_vesion_list">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
   <ul class="skill">
     <li><button>html</button></li>
   <li><button>css</button></li>
```

Comment:
#### Although I highlighted these blocks of code, the changes I am requiring are not for them. I can't highlight the necessary block of codes.

- [ ] The first reviewer highlighted that your mobile design outlook should not be affected by the design for the desktop version. You attempted to make those changes but in the process, I noticed that the change you made affected the outlook and the _**mobile**_ design is not identical to the Figma template. In the **_works_** section, which the first reviewer highlighted, I noticed that I can scroll **_horizontally_** and that affects the design for the _**mobile**_ version. The width of some elements is wider than the **_view width_** for the mobile version. Kindly fix this and make the design not scrollable on the mobile version. 👍 

![chrome-capture-2022-11-17](https://user-images.githubusercontent.com/77106962/208253717-1c17fc9b-329a-40d5-a475-58c286496a7d.gif)

Reply:
- Hey, as stated above by the previous reviewer, your page is still having a horizontal scroll as this issue is yet to be resolved. Please, I wish I could mark this optional as I see you've used all your submission in the dashboard, but i can't because the scroll effect is too obvious to be overlooked and this would not improve the user experience of your website. Please, let's work on this issue to be resolved :pray: